drivers/dai/intel/ssp/ssp.h

```diff
@@ -0,0 +1,384 @@
+/*
+ * Copyright (c) 2022 Intel Corporation.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#ifndef __SOF_DAI_DRIVER_SSP_H__
+#define __SOF_DAI_DRIVER_SSP_H__
+
+#include <stdint.h>
+#include <drivers/dai.h>
+
+#define MASK(b_hi, b_lo)	\
```

Comment:
Lots of macros here that could collide. Can we namespace with ```SSP_INTEL_``` and reuse similar macros from Zephyr common headers. e.g. there are probably macros to set bits etc ?

Reply:
yes will prefix. I only found BIT() from zephyr...